### PR TITLE
[SPARK-25855][CORE][FOLLOW-UP] Format config name to follow the other boolean conf naming convention 

### PR DESCRIPTION
**Pull request by Ngone51**
### What changes were proposed in this pull request?

Change config name from `spark.eventLog.allowErasureCoding` to `spark.eventLog.allowErasureCoding.enabled`.

### Why are the changes needed?

To follow the other boolean conf naming convention.

### Does this PR introduce any user-facing change?

No, it's newly added in Spark 3.0.

### How was this patch tested?

Tested manually and pass Jenkins.

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/SparkConf.scala [hunk 1]
```diff
@@ -700,6 +700,8 @@ private[spark] object SparkConf extends Logging {
     KERBEROS_FILESYSTEMS_TO_ACCESS.key -> Seq(
       AlternateConfig("spark.yarn.access.namenodes", "2.2"),
       AlternateConfig("spark.yarn.access.hadoopFileSystems", "3.0")),
+    EVENT_LOG_ALLOW_EC.key -> Seq(
+      AlternateConfig("spark.eventLog.allowErasureCoding", "3.0")),
```
Comment (HyukjinKwon) [inline core/src/main/scala/org/apache/spark/SparkConf.scala]: `SPARK-25855` seems fixed as of 3.0. I think we can just rename right away.

Comment (Ngone51) [reply core/src/main/scala/org/apache/spark/SparkConf.scala]: Oh..I see.... Thank you for reminding it.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by Ngone51** [comment]
cc @gatorsmile @cloud-fan

**Comment by srowen** [comment]
Don't change this just based on my comment, but "allowErasureCoding.enabled" seems redundant. "erasureCoding.enabled"? doesn't really matter

**Comment by HyukjinKwon** [comment]
Oh I didn't know you commented. For some reasons, "enabled" suffix is being added. I dont particularly like it either but wouldn't it better match it?

**Comment by srowen** [comment]
.enabled seems fine; I'm just saying that "allow..." as well seems redundant then.

**Comment by HyukjinKwon** [comment]
@squito and @vanzin fyi.

**Comment by HyukjinKwon** [comment]
@Ngone51 can you address Sean's comment?

**Comment by Ngone51** [comment]
I think @srowen 's suggestion make sense, I'll address it later.  And it will be better if @squito would also approve on this change.

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Comment by Ngone51** [comment]
Thanks all!

**Pull request closed without merging**